docs/Other SDL Features/Uploading Files/index.md

```diff
@@ -106,10 +106,11 @@ Boolean isPersistent = file.isPersistent();
 Be aware that persistence will not work if space on the head unit is limited. The @![iOS]`SDLFileManager`!@ @![android, javaSE, javaEE]`FileManager`!@ will always handle uploading images if they are non-existent.
 !!!
 
-@![iOS]
+
```

Comment:
in Java Suite overwrite is `true` by default, unlike iOS. So the description for java suite should be updated accordingly